### ISSUE TITLE
Different import paths for run vs freeze

### ISSUE THREAD

**Issue by jkennedy-usgs** [mherrmann/fbs#97]
I'm trying to package my PyQt app on Windows but it seems that modules are imported differently in run mode vs freeze. When calling ```fbs run``` I have all my code in the src/main/python directory, it runs fine. main.py is called first, and other modules in the same directory are imported as needed.

After  calling ```fbs freeze``` however, it seems that the compiled executable is expecting all imported modules to be up one level, in src/main. 

For example, in run mode I can use
```from GSadjust import MainProg```
GSadjust.py is a file in src\main\python containing the class MainProg.

After freezing, I get this error when running the executable:
```
File "python\main.py", line 2, in <module>
ImportError: cannot import name 'MainProg' 
```

If I change the line to
```from python.GSadjust import MainProg```, or if I move GSadjust.py up one level to src\main, it works. main.py is in src/main/python the whole time.

In other words, I can get it to run and freeze, but I have to change the import lines (or file locations) to switch between them.

**Comment by jkennedy-usgs** [comment]
On Mac 10.12.6 I see the same thing, but worse (?)  - ```fbs run``` runs okay, but the app from ```fbs freeze``` expects the imported modules to be _two_ levels up, in the src directory (In the initial comment, on Windows, they needed to be in src/main)

Here is main.py:
```
from fbs_runtime.application_context import ApplicationContext, cached_property
from GSadjust import MainProg
from PyQt5.QtWidgets import QApplication, QSplashScreen
from PyQt5.QtGui import QPixmap, QIcon
import PyQt5.QtCore
import sys, time, logging
			 
class AppContext(ApplicationContext):         
    def run(self):                            
        self.main_window.show()
        return self.app.exec_()               
        
    @cached_property
    def main_window(self):
        return MainProg()

if __name__ == '__main__':
    appctxt = AppContext()                    
    exit_code = appctxt.run()                 
    sys.exit(exit_code)
```

**Comment by mherrmann** [comment]
Hi, not sure why it isn't working, but from my current information I don't think it's a bug in fbs. The reason is that the following work for me:

`main.py`:

    from window import Window
    from fbs_runtime.application_context import ApplicationContext
    
    import sys
    
    class AppContext(ApplicationContext):
        def run(self):
            window = Window()
            window.show()
            return self.app.exec_()
    
    if __name__ == '__main__':
        appctxt = AppContext()
        exit_code = appctxt.run()
        sys.exit(exit_code)

`window.py`:

    from PyQt5.QtWidgets import QMainWindow
    
    class Window(QMainWindow):
        pass

Both `main.py` and `window.py` are in `src/main/python`. `fbs run` and running the output of `fbs freeze` both work fine. Admittedly, I only tested this on Linux now, but don't think there should be difference between OSs with regards to this.

In short, I think it's something specific to your code, rather than a bug in fbs.

**Comment by jkennedy-usgs** [comment]
Thanks for the response, I'll investigate further. Odd that the behavior is different between Windows and Mac. I'll try your simple example and see what happens.

Great project, thanks for your effort.

**Comment by jkennedy-usgs** [comment]
Okay, the problem was the presence of an __init__.py file in the python directory, which apparently keeps the directory off the search path for modules. Not sure why it was there, probably created while trying to solve an earlier problem.

Thanks!
